modules/apps/powwow/powwow-portlet/docroot/meetings/calendar_booking_recurrence_container.jspf

```diff
@@ -86,6 +85,7 @@ if (recurrence != null) {
 <div class="calendar-portlet-recurrence-container hide" id="<portlet:namespace />recurrenceContainer">
 	<table>
 		<aui:input name="exceptionDates" type="hidden" value="<%= exceptionDatesString %>" />
+		<aui:input name="intervalValue" type="hidden" value="<%= interval %>" />
```

Comment:
could we rename this field for better understanding? because its name is `intervalValue` and use value of `interval`, while there is another field named interval `<aui:select label="" name="interval">`. that logic is somehow hard to follow